src/styles/transitions.js

```diff
@@ -65,3 +79,6 @@ export default {
     return Math.round(duration);
   },
 };
+
+export const easing = easingInternal;
```

Comment:
In most cases, theese are referenced via theme.easing. But in some cases (tests, ...) its not possible, so i left it there